examples/apollo/cosmos.proxies.js

```diff
@@ -1,20 +1,10 @@
 import createApolloProxy from 'react-cosmos-apollo-proxy';
 
-const typeDefs = `
-  type Query {
-    hello(who: String): String
-  }
-`;
-
-const mocks = {
-  Query: () => ({
-    hello: (root, { who }) => `Hello ${who ? who : 'C O S M O S'}`
-  })
-};
+// see more examples at:
+// https://github.com/react-cosmos/react-cosmos#react-apollo-graphql
```

Comment:
How about having another example with local mocks? It should also make development easier when testing all types of mocking.

Reply:
Actually, I'm thinking about removing the local mocks part!

GraphQL is all about one source of truth, your schema, and creating local mocks in a separate client app feels like creating a second source of truth, that can easily drift off the alignment with the original API

Reply:
> Actually, I'm thinking about removing the local mocks part!

Makes sense!